Scripts/SDL/Neighborhood.sdl

```diff
@@ -1966,3 +1966,133 @@ STATEDESC Neighborhood
+    VAR BOOL            nb01LinkBookGZVis[1]                    DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BOOL            nb01LinkRoomDoorFunc[1]                 DEFAULT=1   DEFAULTOPTION=VAULT
+    VAR BOOL            nb01RatCreatureVis[1]                   DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BOOL            nb01TelescopeVis[1]                     DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BOOL            nb01WaterfallVis[1]                     DEFAULT=1   DEFAULTOPTION=VAULT
+    VAR BOOL            nb01DRCInfoBoardsVis[1]                 DEFAULT=1   DEFAULTOPTION=VAULT
+    VAR BOOL            nb01YeeshaPage07Vis[1]                  DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BOOL            nb01PlayerImagerVis[1]                  DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BOOL            nb01DRCImagerVis[1]                     DEFAULT=1   DEFAULTOPTION=VAULT
+    VAR BOOL            nb01HappyNewYearVis[1]                  DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BOOL            nb01WebCamVis[1]                        DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BOOL            nb01HoodInfoImagerVis[1]                DEFAULT=1   DEFAULTOPTION=VAULT
+    VAR BOOL            nb01ThanksgivingVis[1]                  DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BOOL            nb01LinkBookNexusVis[1]                 DEFAULT=1   DEFAULTOPTION=VAULT
+    VAR BOOL            nb01Poetry1JournalVis[1]                DEFAULT=1   DEFAULTOPTION=VAULT
+    VAR BOOL            nb01KiNexusJournalVis[1]                DEFAULT=1   DEFAULTOPTION=VAULT
+    VAR BOOL            nb01BahroStonePedestalVis[1]            DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BOOL            nb01BahroPedestalShoutRun[1]            DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BOOL            nb01ReaderBoardVis[1]                   DEFAULT=0   DEFAULTOPTION=VAULT
+
+    VAR STRING32        nb01DRCImagerInbox[1]                               DEFAULTOPTION=VAULT
+
+# Performance variables
+    VAR BOOL            nb01BahroBoatsRun[1]                    DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BYTE            nb01CallSoundChance[1]                  DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BOOL            nb01DarkShapeSwimsRun[1]                DEFAULT=0   DEFAULTOPTION=VAULT
+
+# State variables
+    VAR BYTE            nb01Ayhoheek5Man1State[1]               DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BYTE            nb01CommunityAreaState[1]               DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BYTE            nb01CityLightsArchState[1]              DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BYTE            nb01LakeLightState[1]                   DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BYTE            nb01PrivateRoomsState[1]                DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BYTE            nb01PuzzleWallState[1]                  DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BYTE            nb01PelletUploadState[1]                DEFAULT=0
```

Comment:
```suggestion
    VAR BYTE            nb01PelletUploadState[1]                DEFAULT=0   DEFAULTOPTION=VAULT
```

Reply:
I didnt make this a vault option because I intended it to be age specific and not a global

Reply:
`DEFAULTOPTION=VAULT` doesn't mean the setting is global. It means that the setting in the vault SDL node (whether that be the global SDL node or the age instance SDL node) will override whatever is set in the AgeSDLHook game server state. Shard operators set these kinds of settings in vault SDL nodes, so this is the appropriate flag to use.